StepDefinitions/Hotels/TC18SD.js

```diff
@@ -0,0 +1,39 @@
+const { Given, When, Then,  } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const Homepage = require('../../Pages/Hotels/Homepage');
+
+const homepage = new Homepage();
+
+Then(/^Select “Adults as 6$/, async function () {
+    await homepage.adultPlusButton(4);
```

Comment:
Instead of passing data from SD file try to pass it from feature file like below
```
Then(/^Select “Adults as (.+)$/, async function (number) {
    await homepage.adultPlusButton(number);
```